mempool/src/counters.rs

```diff
@@ -152,25 +157,25 @@ pub fn core_mempool_txn_commit_latency(
 /// Counter tracking latency of txns reaching various stages in committing
 /// (e.g. time from txn entering core mempool to being pulled in consensus block)
 static CORE_MEMPOOL_TXN_COMMIT_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
-    register_histogram_vec!(
+    let histogram_opts = histogram_opts!(
         "aptos_core_mempool_txn_commit_latency",
         "Latency of txn reaching various stages in core mempool after insertion",
-        &["stage", "scope"]
-    )
-    .unwrap()
+        LARGER_LATENCY_BUCKETS.to_vec()
+    );
+    register_histogram_vec!(histogram_opts, &["stage", "scope"]).unwrap()
 });
 
-pub fn core_mempool_txn_ranking_score(stage: &'static str, status: &str, ranking_score: u64) {
+pub fn core_mempool_txn_ranking_bucket(stage: &'static str, status: &str, bucket: &str) {
     CORE_MEMPOOL_TXN_RANKING_SCORE
-        .with_label_values(&[stage, status])
-        .observe(ranking_score as f64);
+        .with_label_values(&[stage, status, bucket])
+        .inc();
 }
 
-static CORE_MEMPOOL_TXN_RANKING_SCORE: Lazy<HistogramVec> = Lazy::new(|| {
-    register_histogram_vec!(
+static CORE_MEMPOOL_TXN_RANKING_SCORE: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
let's add a HistogramVec with high granularity buckets as well, as this is the only visibility we have into the transient gas fee market